### PR TITLE
do not install linphone on machine

### PR DESCRIPTION
**Pull request by fblackburn1**
reason: we now only use docker for linphone

### CODE REVIEW

File: playbooks/wazo-acceptance/pre.yaml [hunk 1]
```diff
@@ -23,15 +22,11 @@
...
-      apt: name={{item}} state=installed
-      with_items:
-        - ansible
-        - libsasl2-dev
-        - linphone-nogtk
-        - python3-dev
-        - lsof
+      apt:
+        name: ansible
+        state: present
```
Comment (bloom1) [inline playbooks/wazo-acceptance/pre.yaml]: isn't that supposed to be `state: installed`?

Comment (fblackburn1) [reply playbooks/wazo-acceptance/pre.yaml]: @fredericlepied what is the inline syntax for a module? where is that documentation?
Also, the module apt do not have the state installed. Why it worked before? and old version of the module?

Comment (fredericlepied) [reply playbooks/wazo-acceptance/pre.yaml]: No idea why it worked. The doc for modules is the regular Ansible documentation.

Comment (fblackburn1) [reply playbooks/wazo-acceptance/pre.yaml]: Ah ok the *inline* syntax is just the regular yaml syntax :sweat_smile:

**Pull request merged**